### DISCUSSION TITLE
can use alias

### DISCUSSION THREAD

**Discussion by hosamalzagh** [sevdokimov/FriendlyTerminal#8] [Q&A]
hello 
is support alias ?

**Comment by sevdokimov** [comment]
Could you clarify what do you mean?

**Comment by hosamalzagh** [comment]
alias pa="php artisan"

alias pam="php artisan migrate"
alias pam:s="php artisan migrate:status"
alias pam:roll="php artisan migrate:roll"
alias pam:s='php artisan migrate --seed'
alias pam:rs='php artisan migrate:refresh --seed'
alias pa:db="php artisan db:seed"

alias pa:k="php artisan key:generate"
